pkg/storage/segments/repairer.go

```diff
@@ -122,9 +142,17 @@ func (repairer *Repairer) Repair(ctx context.Context, path storj.Path) (err erro
 		return Error.Wrap(err)
 	}
 
+	var requestCount int
```

Comment:
style nit:  I personally would not add three lines of code to clarify the scope of one variable

Reply:
it was in one line before, but there was a comment to break it down in 2 steps and I couldn't disagree that it's clearer.